### ISSUE TITLE
alliance playbook setup - psycopg2 error on Windows

### ISSUE THREAD

**Issue by cjohnsonjava** [NorthBridge/alliance-community#28]
When running pip install -r requirements.txt in cmd
received psycopg2 failed with error code 1

**Comment by hectron** [comment]
Hi @cjohnsonjava , can you please provide more information?

Any additional information you can provide, steps you took, etc would be helpful. Some information that would be necessary:

**Windows Version:**
**32/64 Bit:**
**PostgreSQL Version:**
**Python Version:**
**Pip Version:**
**Virtualenv version:**
**Steps to reproduce:**
**Stack trace:**

**Comment by cjohnsonjava** [comment]
Hi @hectron here is the information you requested

Windows Version: Windows 7 Home Premium
32/64 Bit: 64 bit
PostgreSQL Version: 9.4
Python Version: 2.7.9
Pip Version: 7.1.2
Virtualenv version: 13.1.0

To get the Virtualenv version, in the command prompt I typed in the command virtualenv --version

Steps to reproduce:  Install python, PostgreSQL and also make sure the Environment Variables in Windows are setup
Environment Variables
1.	Go to your environment variables (Control Panel--> search: environment variables --> edit environment variables):
a.	In system variables, “Path”  add 
   ;%PYTHON_HOME%;%PYTHON_HOME%/Scripts;
b.	In user variables, “Path” add C:\Program Files\PostgreSQL\9.4\bin;

2.	Get the project code(See instructions under step 4 in this link https://github.com/rebeldroid12/alliance-community/blob/master/docs/Windows_Install_Instructions.md

3.	Install project dependencies
a.	Install pip. Download get-pip.py and save (this will probably go into your Downloads Folder)
         https://raw.githubusercontent.com/pypa/pip/master/contrib/get-pip.py
      Once downloaded go into the command prompt:
         cd ~/Downloads
         python get-pip.py


b.	In the command prompt:
	 pip install virtualenvwrapper-win

c.	Go to the project directory: Make sure you are in the directory alliance-community
	  mkvirtualenv alliance

d.	Run the command: pip install -r requirements.txt

After running pip install -r requirements.txt, I receive the psycopg2 error

To try to resolve the issue, I performed the following steps

1.)	Downloaded VCForPython27.msi - http://aka.ms/vcpython27
2.)	Next in the command prompt as an administrator, I ran the command      msiexec /i %USERPROFILE%\Downloads\VCForPython27.msi ALLUSERS=1
3.)	Next, I went to the alliance virtual environment and also ran pip install -r requirements.txt as an administrator.

After running pip install -r requirements.txt, I received the psycopg2 error again.

Stack trace: C:\Users\CorettaDM\Envs\alliance\Scripts\python.exe -c "import setupto
ls, tokenize;__file__='c:\\users\\corett~1\\appdata\\local\\temp\\pip-build-hlw
xe\\psycopg2\\setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).
ead().replace('\r\n', '\n'), __file__, 'exec'))" install --record c:\users\core
t~1\appdata\local\temp\pip-t_cynd-record\install-record.txt --single-version-ex
ernally-managed --compile --install-headers C:\Users\CorettaDM\Envs\alliance\in
lude\site\python2.7\psycopg2" failed with error code 1 in c:\users\corett~1\appata\local\temp\pip-build-hlwixe\psycopg2

**Comment by hectron** [comment]
After so much work, we were finally able to resolve this.

The absolute reason why the program was not installing was because of an issue with VC for Python (a Windows program). I'm not entirely sure if this has anything to do with Visual C for Windows, but I believe that it had something to do with that.

We ended up manually downloading `psycopg2` for **Windows 32 Bit** and installed it in her virtualenvironment using `easy_install`. Once that was done, we were able to get her database set up and have the application load. However, the state of the master branch was broken at the time we tried.

I want to say that this is now working. The tests that Brad worked on earlier ran fine.
